notebooks/ImvizExample.ipynb

```diff
@@ -92,69 +102,61 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "acs_47tuc_1 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03gjq_flc.fits', cache=True)\n",
-    "acs_47tuc_2 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03h1q_flc.fits', cache=True)"
+    "files = ['jw02731-o001_t017_nircam_clear-f090w_i2d.fits',\n",
+    "         #'jw02731-o001_t017_nircam_clear-f187n_i2d.fits',\n",
```

Comment:
Should we even give so many in the list? Someone is going to try to uncomment everything and their machine is going to 🔥

Reply:
For the record, I only see small performance issues when loading and viewing all of these files.  And hopefully those will be improved soon.  I think it's useful to have them for anyone who wants to try, but maybe a warning saying they're large files and performance may be affected depending on system specs?

Reply:
I'm updating to use the Cartwheel images instead, which are ~600 MB instead of 3.2 GB.